### DISCUSSION TITLE
NvChad and meson

### DISCUSSION THREAD

**Discussion by MeGreen** [NvChad/NvChad#2392] [Q&A]
Is there a plugin or way to integrate meson, so that the lsp doesn't complain that librarys are not found, if they are correctly defined as dependencies in meson.build?

**Comment by siduck** [comment]
no idea.  ask on r/neovim

**Comment by mmapro12** [comment]
Error detected while processing /home/muhammed/.config/nvim/init.lua:
E5113: Error while calling lua chunk: /home/muhammed/.config/nvim/init.lua:7: attempt to index field 'uv' (a nil value)
stack traceback:
        /home/muhammed/.config/nvim/init.lua:7: in main chunk
Press ENTER or type command to continue

I get this error when i try to install nvchad to Ubuntu

**Reply by siduck** [reply]
@mmapro12  use latest stable version of nvim which is 0.10.3  . for ubuntu there's an unstable ppa
